src/main/java/org/elasticsearch/bootstrap/Bootstrap.java

```diff
@@ -92,6 +91,16 @@ public boolean handle(int code) {
             });
         }
     }
+    
+    private void setupSecurity(Settings settings, Environment environment) throws Exception {
+        ESLogger logger = Loggers.getLogger(Bootstrap.class);
+        if (settings.getAsBoolean("security.enabled", true)) {
+            Security.configure(environment);
+            logger.info("security enabled");
+        } else {
+            logger.warn("security disabled");
+        }
```

Comment:
can we debug log the default? also leaning to have info the "non default" setting, thats what we try to do most times in other components to try and keep the startup logs clean and informative.